nightmares/temp.test.js

```diff
@@ -0,0 +1,17 @@
+var Nightmare = require('nightmare');
+var nightmare = Nightmare({ show: process.env.SHOW_ELECTRON || false, waitTimeout: 3000 });
+
+describe('Should open redux test', function () {
+	this.timeout(10000);
+	it('should pass', function (done) {
```

Comment:
rather than calling this with done, you can just return the promise from inside the it block.